### PR TITLE
Remove unnecessary placement rules from OrchardCore.Content

### LINKED ISSUE #16028: Unnecessary placement rules in OrchardCore.Contents
Currently in OrchardCore.Contents module there is  a placement rule in placements. json as so : 

```
 "Content_PublishButton": [
    {
      "place": "Actions:5" // Same as in the driver
    }
  ], 

```
Apart from that the placement is already set from the driver this prevents other modules to add there own placement rule for this shape unless they take a dependency on this module. 

Also there  is a placement rule : 

```
  "Content_SaveButton": [
    {
      "place": "Actions:6" // Move the save button after the publish
    }
  ],

```

that is not used anywhere as there is no shape with this name.
 I propose to remove these 2 rules.

### PR DESCRIPTION
**Pull request by giannik**
Fixes #16028

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.Contents/placement.json [hunk 1]
```diff
@@ -6,16 +6,6 @@
 */
 
 {
-  "Content_PublishButton": [
-    {
-      "place": "Actions:5" // Same as in the driver
```
Comment (Piedone) [inline src/OrchardCore.Modules/OrchardCore.Contents/placement.json:11]: Are you sure the driver's the one taking effect with `Action:10`?

**Review by Piedone — CHANGES_REQUESTED** [review]

File: src/OrchardCore.Modules/OrchardCore.Contents/placement.json (hunk 1 above)
Comment (giannik) [reply src/OrchardCore.Modules/OrchardCore.Contents/placement.json:11]: apart from this line : 
https://github.com/OrchardCMS/OrchardCore/blob/0c00316259d47f2c7abf084d3fc7b0d7f5a48ab9/src/OrchardCore.Modules/OrchardCore.Contents/Drivers/ContentsDriver.cs#L104

where else would it be used  ?

Comment (Piedone) [reply src/OrchardCore.Modules/OrchardCore.Contents/placement.json:11]: I mean, are you sure that when the layout is rendered with the current `main` code, then the placement that will take effect is the one defined in the driver, with `Action:10`, and not this one with `Action:5`?

Comment (giannik) [reply src/OrchardCore.Modules/OrchardCore.Contents/placement.json:11]: I tested in the admin **edit content page** and **widget edit page** and it works the same when applying this pr.

image


image

Comment (Piedone) [reply src/OrchardCore.Modules/OrchardCore.Contents/placement.json:11]: OK, thank you!

**Review by Piedone — APPROVED** [review]

**Pull request merged**